core/storage/question/gae_models.py

```diff
@@ -271,8 +272,8 @@ def get_question_skill_links_based_on_difficulty_equidistributed_by_skill(
             total_question_count: int. The number of questions expected.
             skill_ids: list(str). The ids of skills for which the linked
                 question ids are to be retrieved.
-            difficulty_requested: int. The number of the difficulty requested
-                to be fetched.
+            difficulty_requested: float. The skill difficulty of the questions
```

Comment:
Just for sanity-checking ... it really is a float, right? (I did think it was 0 ... 1 but I wasn't 100% sure, so I just want to make sure that you've actually verified this and that we aren't somehow passing the index to a list of difficulties or something like that.)

Also, if it is indeed a float in [0, 1], then why did you put int before?

Reply:
Yeah it is a float. It was a mistake before. Sorry about that.